src/components/NcPopover/NcPopover.vue

```diff
@@ -157,8 +234,36 @@ export default {
 	emits: [
 		'after-show',
 		'after-hide',
+		/**
+		 * @see https://floating-vue.starpad.dev/api/#update-shown
+		 */
+		'update:shown',
 	],
 
+	data() {
+		return {
+			internalShown: this.shown,
+		}
+	},
+
+	computed: {
+		shownProxy: {
+			get() {
+				return this.internalShown
```

Comment:
@ShGKme  I had to adjust this from `shown` to `internalShown` to make it work. See https://github.com/nextcloud-libraries/nextcloud-vue/pull/5086/files#r1458568969.